test/example_schema.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
     username: {
       type: 'string',
       min: 3,
-      max: 20
+      max: 20,
+      individual: true
```

Comment:
@nelsonic Good to call upon, check PR description but will add what I said to the issue now.

Essentially since we blindly try to copy the joi api where possible in create joi schemas we need to use different method names